BaseApi.js

```diff
@@ -173,6 +173,12 @@ const API = {
     const url = ct_url.replace('$objectPk', objectPk);
     return makePostRequest(url, data, token);
   },
+  editComment(contentTypeId, objectPk, commentPk, data, token = null) {
```

Comment:
root level comment editing does not work for me:
<img width="1083" alt="Bildschirmfoto 2022-06-30 um 12 17 14" src="https://user-images.githubusercontent.com/52801593/176664420-a5716955-6fcc-440c-b392-ab2caf859a26.png">

child level comment editing works fine!

Reply:
fixed, thanks!